code/modules/clothing/spacesuits/rig.dm

```diff
@@ -101,6 +151,7 @@
 	species_fit = list("Vox")
 	_color = "syndi"
 	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 35, bio = 100, rad = 60)
+	action_button_name = "Toggle Helmet Camera" //This helmet does not have a light, but we'll do as if

```

Comment:
I preferred a dynamic proc that allows one to dynamically remove a helmet's light ability

Just setting the gold helmet's action button to null would work, but if a new helmet with no light was added in, it would need that update. Also it's basic framework to allow disabling helmet lights